test/functionalTest/cases/0950_GET_v2_entity/GET_v2_entity.test

```diff
@@ -95,37 +95,37 @@ Date: REGEX(.*)
             "contextElement": {
                 "attributes": [
                     {
-                        "name": "speed",
```

Comment:
"brand": "Mercedes Benz" 

    has been changed to 

    "brand": {
       "type": "string", 
       "value": "Mercedes Benz"
    }

etc.  No, there are a few changes in the file, not only spaces (the spaces probably come from copying the contents of the out file of the functest)

NTC